src/m365/spo/commands/list/list-get.spec.ts

```diff
@@ -1281,4 +1281,24 @@ describe(commands.LIST_GET, () => {
     const actual = await command.validate({ options: { webUrl: 'https://contoso.sharepoint.com', id: '0CD891EF-AFCE-4E55-B836-FCE03286CCCF' } }, commandInfo);
     assert(actual);
   });
-});
+
+  it('retrieves the default list in the specified site by providing a webUrl', async () => {
+    const defaultSiteList = { ...listResponse, BaseTemplate: 101, ParentWebUrl: "/", ListItemEntityTypeFullName: "SP.Data.Shared_x0020_DocumentsItem" };
+
+    sinon.stub(request, 'get').resolves({
```

Comment:
Let's check that the right URL was passed.

Reply:
@milanholemans What do you mean by saying this? I mean we are testing if a user only passes the webUrl param. Then – and only then – the default document library will be returned.
Probably I don't understand your id on checking that the right URL was passed?

Reply:
Just like in other tests, instead of a `sinon.stub(...).resolves(...)`, we should do `sinon.stub(...).callsFake(opts => if (opts.url === <URL>) return result;`. This way we ensure that the right request was made.

Reply:
Done.